content/agent/prometheus.md

```diff
@@ -69,9 +69,9 @@ If you have more advanced needs than the generic check (have some metrics prepro
 AND/OR
 - Create method named after the prometheus metric they will handle (see `self.prometheus_metric_name`)
 
-## Writting a custom Prometheus check
+## Writing a custom Prometheus check
 
-This is a simple example of writting a kube DNS check to illustrate the `PrometheusCheck` class usage for learning purposes, but the example bellow could have been handled with configuring the following generic prometheus check:
+This is a simple example of writing a kube DNS check to illustrate the `PrometheusCheck` class usage for learning purposes, but the example bellow could have been handled with configuring the following generic prometheus check:
```

Comment:
* "below" has one L, not two.
* Capitalise Prometheus.
* Break this into two sentences
> This is a simple example … for learning purposes. The example below could …